docs/integrations/community-ecosystem-apps.md

```diff
@@ -23,19 +23,75 @@ The following links provide collection, installation, and dashboard information
 
 More information on this initiative and how to contribute can be found [here](#faq).
 
-## Community Content App Catalog
+## Community Content Catalog
 
-[Add info and screenshots showing how to find Community apps]
+Community content is stored in our [Community Content Repository](https://github.com/SumoLogic/sumologic-content). Please visit the following links to access related content:
```

Comment:
@wjakelee , do you think this section might look better as a bulleted list?

Also, thanks for putting this together!

Reply:
Yeah I think you are correct. Probably better as bulleted list!